test/runtests.jl

```diff
@@ -37,6 +37,7 @@ if group == "All" || group == "Interface"
 end
 
 if group == "All" || group == "Integrators"
+    @time @testset "Adams Variable Coefficients Tests" begin include("ode/adams_tests.jl") end
```

Comment:
I think that this line belongs to the `"AlgConvergence"` group.

Reply:
in `adams_test.jl` there are only tests for `coefs`, not for `convergence`

Reply:
It definitely doesn't make sense in integrators though since it has nothing to do with the integrators. It is a unit test for convergence.